tests/cluster/delta/test_delta_class.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+
+from pyspark.sql.utils import AnalysisException
+
+from atc.config_master import TableConfigurator
+from atc.delta import DbHandle, DeltaHandle
+from atc.spark import Spark
+
+
+class DeltaTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        TableConfigurator().clear_all_configurations()
+
+    def test_01_configure(self):
+        tc = TableConfigurator()
+        tc.register(
+            "MyDb", {"name": "TestDb{ID}", "path": "/mnt/atc/silver/testdb{ID}"}
+        )
+
+        tc.register(
+            "MyTbl",
+            {
+                "name": "TestDb{ID}.TestTbl",
+                "path": "/mnt/atc/silver/testdb{ID}/testtbl",
+            },
+        )
+
+        # test instantiation without error
+        DbHandle.from_tc("MyDb")
+        DeltaHandle.from_tc("MyTbl")
+
+    def test_02_write(self):
```

Comment:
Do you want to test, that overwrite of schema actually works? 

Idea:

First write some data with:
df = Spark.get().createDataFrame([(1, "a"), (2, "b")], "id int, name string")
dh.overwrite(df, overwriteSchema)
assert that data is as expected

then use overwrite:
df = Spark.get().createDataFrame([(1, "a","yes"), (2, "b", "no")], "id int, name string, response string").
dh.overwrite(df, overwriteSchema)
assert that data is as expected

Reply:
agreed.